### ISSUE TITLE
Import pod fails during import of containerDisk

### ISSUE THREAD

**Issue by ksimon1** [kubevirt/containerized-data-importer#1453]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
During import of container disk, import pod fails
```
    oc logs -n kubevirt -f importer-windows10-datavolume-original
    I1022 06:31:02.635724       1 importer.go:52] Starting importer
    I1022 06:31:02.635831       1 importer.go:117] begin import process
    I1022 06:31:02.635846       1 data-processor.go:305] Calculating available size
    I1022 06:31:02.635885       1 data-processor.go:317] Checking out file system volume size.
    I1022 06:31:02.635894       1 data-processor.go:325] Request image size not empty.
    I1022 06:31:02.635909       1 data-processor.go:330] Target size 63073701888.
    I1022 06:31:02.635973       1 util.go:37] deleting file: /scratch/lost+found
    I1022 06:31:02.637099       1 util.go:37] deleting file: /data/lost+found
    I1022 06:31:02.637442       1 data-processor.go:227] New phase: TransferScratch
    I1022 06:31:02.637466       1 registry-datasource.go:84] Copying registry image to scratch space.
    I1022 06:31:02.637474       1 transport.go:214] Downloading image from docker://quay.io/openshift-cnv/ci-common-templates-images:windows10, copying file from disk to /scratch
    I1022 06:31:03.410360       1 transport.go:238] Processing layer {Digest:sha256:f0caa32766f3a1f4976dd796574e11883b0edba0274143c47dba3a08b100c14f Size:9283647666 URLs:[] Annotations:map[] MediaType:application/vnd.docker.image.rootfs.diff.tar.gzip CompressionOperation:0 CompressionAlgorithm:<nil> CryptoOperation:0}
    I1022 06:31:04.030823       1 transport.go:184] File 'disk/win10Updated.qcow2' found in the layer
    E1022 06:36:02.637799       1 transport.go:109] Error copying file: context deadline exceeded
    E1022 06:36:02.637848       1 transport.go:199] Could not copy file to scratch space: Error retrieving image: context deadline exceeded
    E1022 06:36:02.637871       1 transport.go:252] Failed to find VM disk image file in the container image
    E1022 06:36:02.637890       1 data-processor.go:224] Failed to find VM disk image file in the container image
    kubevirt.io/containerized-data-importer/pkg/image.copyRegistryImage
            pkg/image/transport.go:253
    kubevirt.io/containerized-data-importer/pkg/image.CopyRegistryImage
            pkg/image/transport.go:268
    kubevirt.io/containerized-data-importer/pkg/importer.(*RegistryDataSource).Transfer
            pkg/importer/registry-datasource.go:85
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
            pkg/importer/data-processor.go:181
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
            pkg/importer/data-processor.go:156
    main.main
            cmd/cdi-importer/importer.go:172
    runtime.main
            GOROOT/src/runtime/proc.go:203
    runtime.goexit
            src/runtime/asm_amd64.s:1357
    Failed to read registry image
    kubevirt.io/containerized-data-importer/pkg/importer.(*RegistryDataSource).Transfer
            pkg/importer/registry-datasource.go:87
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
            pkg/importer/data-processor.go:181
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
            pkg/importer/data-processor.go:156
    main.main
            cmd/cdi-importer/importer.go:172
    runtime.main
            GOROOT/src/runtime/proc.go:203
    runtime.goexit
            src/runtime/asm_amd64.s:1357
    Unable to transfer source data to scratch space
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
            pkg/importer/data-processor.go:186
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
            pkg/importer/data-processor.go:156
    main.main
            cmd/cdi-importer/importer.go:172
    runtime.main
            GOROOT/src/runtime/proc.go:203
    runtime.goexit
            src/runtime/asm_amd64.s:1357
    I1022 06:36:04.093755       1 util.go:37] deleting file: /scratch/disk
    E1022 06:36:04.093892       1 importer.go:174] Failed to find VM disk image file in the container image
    kubevirt.io/containerized-data-importer/pkg/image.copyRegistryImage
            pkg/image/transport.go:253
    kubevirt.io/containerized-data-importer/pkg/image.CopyRegistryImage
            pkg/image/transport.go:268
    kubevirt.io/containerized-data-importer/pkg/importer.(*RegistryDataSource).Transfer
            pkg/importer/registry-datasource.go:85
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
            pkg/importer/data-processor.go:181
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
            pkg/importer/data-processor.go:156
    main.main
            cmd/cdi-importer/importer.go:172
    runtime.main
            GOROOT/src/runtime/proc.go:203
    runtime.goexit
            src/runtime/asm_amd64.s:1357
    Failed to read registry image
    kubevirt.io/containerized-data-importer/pkg/importer.(*RegistryDataSource).Transfer
            pkg/importer/registry-datasource.go:87
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
            pkg/importer/data-processor.go:181
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
            pkg/importer/data-processor.go:156
    main.main
            cmd/cdi-importer/importer.go:172
    runtime.main
            GOROOT/src/runtime/proc.go:203
    runtime.goexit
            src/runtime/asm_amd64.s:1357
    Unable to transfer source data to scratch space
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
            pkg/importer/data-processor.go:186
    kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
            pkg/importer/data-processor.go:156
    main.main
            cmd/cdi-importer/importer.go:172
    runtime.main
            GOROOT/src/runtime/proc.go:203
    runtime.goexit
            src/runtime/asm_amd64.s:1357
```
**What you expected to happen**:
Import pod finishes import of container disk
**How to reproduce it (as minimally and precisely as possible)**:
1) Download windows10 image
2) Run command `virt-sparsify win_10.qcow2 win10Updated.qcow2 --check-tmpdir=ignore`
3) Create container disk
```
FROM scratch
ADD win10Updated.qcow2 /disk/
```
4) Push it to some registry
5) Create dataVolume
```
oc apply -n kubevirt -f - <<EOF
apiVersion: cdi.kubevirt.io/v1beta1
kind: DataVolume
metadata:
  name: windows10-datavolume-original
spec:
  source:
    registry:
      url: "<URL-REGISTRY>"
  pvc:
    accessModes:
      - ReadWriteOnce
    resources:
      requests:
        storage: 60Gi
EOF
```
**Anything else we need to know?**:

**Environment**:
- CDI version (use `kubectl get deployments cdi-deployment -o yaml`): v1.24.2

**Comment by awels** [comment]
@tomob Can you take a look at this?

**Comment by tomob** [comment]
This should be included already in 1.23.7, as it has been fixed with PR #1413

**Comment by awels** [comment]
Great, I just didn't backport it to 1.24 branch. Will be in 1.25 when I release it.
